src/ImageGallery.jsx

```diff
@@ -582,11 +582,15 @@ export default class ImageGallery extends React.Component {
 
             onTouchStart={event => this.slideToIndex.call(this, index, event)}
             onClick={event => this.slideToIndex.call(this, index, event)}>
-
-            <img
-              src={item.thumbnail}
-              alt={item.thumbnailAlt}
-              onError={onThumbnailError.bind(this)}/>
+            <div style={{ width: '63px' }}>
```

Comment:
I forgot to mention, don't use arbitrary width like `63px` the label should be responsive with the parent container.